TypeCobol/Compiler/Scanner/Scanner.cs

```diff
@@ -470,7 +471,7 @@ public static void ScanTokensLineContinuationGroup(IList<TokensLine> continuatio
                             if (lastTokenOfConcatenatedLineSoFar.HasClosingDelimiter)
                             {
                                 bool continuationStartsWithTwoDelimiters = ((startOfContinuationIndex + 1) <= lastIndex && line[startOfContinuationIndex + 1] == lastTokenOfConcatenatedLineSoFar.ExpectedClosingDelimiter);
-                                bool previousDelimiterIsNotAtEnd = (lastTokenOfConcatenatedLineSoFar.EndColumn + CobolFormatAreas.Indicator) != CobolFormatAreas.End_B;
+                                bool previousDelimiterIsNotAtEnd = lastTokenOfConcatenatedLineSoFar.EndColumn != (int) CobolFormatAreas.End_B;
```

Comment:
Could you please explain this modification?

Reply:
Before:
The continuation line was the content of column 8 to 72 from the original line.
So the last column of the continuation line was 65.
That's why `CobolFormatAreas.Indicator` (value =7) was added to the the column of the last token to check if it ended at column 72.

After:
The continuation line is the content of column 1 to 72 from the original line.
So the last column of the continuation line is now also 72.
There is no need to add `CobolFormatAreas.Indicator` anymore.